app/Resources/views/article.html.twig

```diff
@@ -316,6 +316,68 @@
 
     {%- endif %}
 
+    <script type="application/ld+json">
+        {
+            "@context": "https://schema.org",
+            "@type": "ScholarlyArticle",
+            "mainEntityOfPage": {
+                "@type": "WebPage",
+                "@id": "{{ url('article', {id: item.id}) }}"
+            },
+            "headline": "{{ item.fullTitle|striptags }}",
+            "datePublished": "{{ item.publishedDate|date('Y-m-d', 'UTC') }}",
+            "author": [
+                {% for author in item.authors %}
+                    {% if author.orcid is defined %}
+                        {
+                            "@type": "Person",
+                            "name": "{{ author.preferredName }}"
+                        }
+                    {% elseif author.people is defined %}
+                        {
+                            "@type": "Organization",
+                            "name": "{{ author.toString() }}"
+                        }
+                    {% else %}
+                        "{{ author.toString() }}"
+                    {% endif %}
+                    {%- if not loop.last %}, {% endif -%}
+                {% endfor %}
+            ],
+            "publisher": {
+                "@type": "Organization",
+                "name": "eLife Sciences Publications, Ltd",
+                "logo": {
+                "@type": "ImageObject",
+                "logo": "{{ absolute_url(asset('assets/patterns/img/patterns/organisms/elife-logo-full.svg')) }}"
+                }
+            },
+            {% if item.keywords is defined and item.keywords|length %}
+                "keywords": [
+                    {% for keyword in item.keywords %}
+                        "{{ keyword|striptags }}"
+                        {% if not loop.last %},{% endif %}
+                    {% endfor %}
+                ],
+            {% endif %}
+            {% if item.subjects|length %}
+                "about": [
+                    {% for subject in item.subjects %}
+                    "{{ subject.name|striptags }}"
+                    {% if not loop.last %},{% endif %}
+                    {% endfor %}
+                ],
+            {% endif %}
+            "isPartOf": {
+                "@type": "Periodical",
+                "name": "eLife",
+                "issn": "2050-084X"
+            }
+            {% if item.impactStatement|default(null) -%},
```

Comment:
Suggest this goes above `isPartOf` so the comma is not part of the `if` statement.